generated/language/src/main/java/com/google/cloud/language/v1/spring/LanguageServiceSpringProperties.java

```diff
@@ -0,0 +1,469 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.language.v1.spring;
+
+import com.google.api.core.BetaApi;
+import com.google.cloud.spring.core.Credentials;
+import com.google.cloud.spring.core.CredentialsSupplier;
+import javax.annotation.Generated;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+import org.threeten.bp.Duration;
+
+// AUTO-GENERATED DOCUMENTATION AND CLASS.
+/** Provides default property values for LanguageService client bean */
+@Generated("by gapic-generator-java")
+@BetaApi("Autogenerated Spring autoconfiguration is not yet stable")
+@ConfigurationProperties("com.google.cloud.language.v1.spring.auto.language-service")
+public class LanguageServiceSpringProperties implements CredentialsSupplier {
+  @NestedConfigurationProperty
+  private final Credentials credentials =
+      new Credentials(
+          "https://www.googleapis.com/auth/cloud-language",
+          "https://www.googleapis.com/auth/cloud-platform");
+
+  private String quotaProjectId;
+  private Integer executorThreadCount;
+  private boolean useRest = false;
+  private Duration analyzeSentimentInitialRetryDelay;
```

Comment:
As mentioned offline, a lot of duplication here can probably be avoided by creating an object for retry settings.

Reply:
A helper object sounds like a good idea.